locals.tf

```diff
@@ -0,0 +1,6 @@
+locals {
+  lambda_buckets = {
```

Comment:
Lets add a comment here that additional regions can be added by contacting support.